mythril/interfaces/cli.py

```diff
@@ -389,6 +389,7 @@ def main():
                     max_depth=args.max_depth,
                     execution_timeout=args.execution_timeout,
                     create_timeout=args.create_timeout,
+                    transaction_count=1,
```

Comment:
Why is it 1 here by default?

Reply:
Also I'd say that for the graph functionality it doesn't really make sense to do multi transactional analysis.
I'd think that symbolic storage is fine here.
What do you think?

Reply:
```
Also I'd say that for the graph functionality it doesn't really make sense to do multi transactional analysis.
```

This answers your first question - that's why I set it to the fixed value 1 instead of args.transaction_count. Does that make sense?